docs/.vitepress/config.js

```diff
@@ -34,81 +34,108 @@ export default defineConfig({
+                            {
+                                text: 'Endpoints', collapsed: true, link: '/api-reference/endpoints/',
+                                items: [
+                                    {
+                                        text: 'Directions', collapsed: true, link: '/api-reference/endpoints/directions/',
+                                        items: [
+                                            {text: 'Requests and Return Types', link: '/api-reference/endpoints/directions/requests-and-return-types'},
+                                            {text: 'Routing Options', link: '/api-reference/endpoints/directions/routing-options'},
+                                            {
+                                                text: 'Extra info', collapsed: true, link: '/api-reference/endpoints/directions/extra-info/',
+                                                items: [
+                                                    {text: 'Steepness IDs', link: '/api-reference/endpoints/directions/extra-info/steepness'},
+                                                    {text: 'Surface IDs', link: '/api-reference/endpoints/directions/extra-info/surface'},
+                                                    {text: 'Category IDs', link: '/api-reference/endpoints/directions/extra-info/waycategory'},
+                                                    {text: 'Type IDs', link: '/api-reference/endpoints/directions/extra-info/waytype'},
+                                                    {text: 'Difficulty IDs', link: '/api-reference/endpoints/directions/extra-info/trail-difficulty'},
+                                                    {text: 'Restriction IDs', link: '/api-reference/endpoints/directions/extra-info/road-access-restrictions'},
+                                                    {text: 'Country IDs', link: '/technical-details/country-list'},
+                                                ]
+                                            },
+                                            {text: 'Route Attributes', link: '/api-reference/endpoints/directions/route-attributes'},
+                                            {text: 'Geometry Decoding', link: '/api-reference/endpoints/directions/geometry-decoding'},
+                                            {text: 'Instruction Types', link: '/api-reference/endpoints/directions/instruction-types'},
+                                        ]
+                                    },
+                                    {text: 'Isochrones', link: '/api-reference/endpoints/isochrones/'},
+                                    {text: 'Matrix', link: '/api-reference/endpoints/matrix/'},
+                                    {text: 'Snapping', link: '/api-reference/endpoints/snapping/'},
+                                    {text: 'Export', link: '/api-reference/endpoints/export/'},
```

Comment:
Maybe we could leave them in the doc (for people running the ORS themselfes), but clearly mark them as not part of our live API? In the side bar and on the pages themselfes?

Reply:
Yes, sounds good :)
To me, a clearly marked comment on the page itself (e.g. like the blue info boxes in confluence) should suffice, but we could also mention sth in the sidebar.

Reply:
I like the idea with the info boxes - changed it accordingly, thanks!

Reply:
I think we want to activate snapping on our live API.
The export endpoint should be marked as experimental, as it will probably change a lot in the future.